articles/quickstart/spa/angular2/04-authorization.md

```diff
@@ -115,11 +109,13 @@ export const ROUTES: Routes = [
 ];
 ```
 
-The guard implements the `CanActivate` interface which requires a method called `canActivate` in the service. This method returns `true` if the user is authenticated and `false` if not. It also navigates the user to the home route if they aren't authenticated.
+The guard implements the `CanActivate` interface which requires a method called `canActivate` in the service. This method returns `true` if the user is authenticated and `false` if they are not. It also navigates the user to the home route if they are not authenticated.
```

Comment:
Just a thought, could the interface have a different name? The difference between the interface name and the method name is just a capital 'C'. Wouldn't it potentially cause problems to spot errors in code?